core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -230,6 +234,10 @@ func (f *FwdMgr) runLoop() {
 				continue
 			}
 
+			if err := f.logpoller.Ready(); err != nil {
+				f.logger.Warnw("Skipping log syncing", "err", err)
```

Comment:
Does this one need to `continue`?
```suggestion
				f.logger.Warnw("Skipping log syncing", "err", err)
				continue
```

Reply:
Nice catch!